orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/kato/tasks/quip/TriggerQuipTaskSpec.groovy

```diff
@@ -250,7 +253,7 @@ class TriggerQuipTaskSpec extends Specification {
 
     then:
     2 * instanceService.getCurrentVersion(app) >> {
-      throw RetrofitError.networkError('http://foo', new IOException('failed'))
+      throw new SpinnakerNetworkException(new RetrofitError(null, null, null, null, null, null, null))
```

Comment:
This would be a smaller change...

```suggestion
      throw new SpinnakerNetworkException(RetrofitError.networkError('http://foo', new IOException('failed')))
```

Reply:
Modified the wrapper exception from generic RetrofitError to more specific Retrofit network error